modules/core/src/v2/wallet.ts

```diff
@@ -1414,7 +1414,7 @@ export class Wallet {
         });
 
         if (derivedKeypair) {
-          addressParams.derivedAddress = derivedKeypair.pub;
+          addressParams.derivedAddress = derivedKeypair.address ?? derivedKeypair.pub;
```

Comment:
I created this PR https://github.com/BitGo/BitGoJS/pull/1952, it should help to make the code more coin agnostic